pyproject.toml

```diff
@@ -154,3 +154,38 @@ files = "src"
 explicit_package_bases = true
 mypy_path = "$MYPY_CONFIG_FILE_DIR/src"
 namespace_packages = true
+
+[tool.towncrier]
+package = "ansys.openapi.common"
+directory = "doc/changelog.d"
+filename = "CHANGELOG.md"
+start_string = "<!-- towncrier release notes start -->\n"
+underlines = ["", "", ""]
+template = "doc/changelog.d/changelog_template.jinja"
+title_format = "## [{version}](https://github.com/ansys/openapi-common/releases/tag/v{version}) - {project_date}"
+issue_format = "[#{issue}](https://github.com/ansys/openapi-common/pull/{issue})"
+
+[[tool.towncrier.type]]
+directory = "added"
+name = "Added"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "changed"
+name = "Changed"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "fixed"
+name = "Fixed"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "dependencies"
+name = "Dependencies"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "miscellaneous"
+name = "Miscellaneous"
+showcontent = true
```

Comment:
Shouldn't these match the labels that we use in the repo?

Reply:
Oh right, there is a hardcoded mapping in the action https://github.com/ansys/actions/blob/7c46e54da0e3d73141e79f41a63a43a24cb855b8/doc-changelog/action.yml#L120

Reply:
Hmm, these are the headings in the changlog, and they're the ones that the PyAnsys team recommends, they have a map you can see here:

https://github.com/ansys/actions/blob/7c46e54da0e3d73141e79f41a63a43a24cb855b8/doc-changelog/action.yml#L120-L125